sub-packages/bionemo-scdl/src/bionemo/scdl/io/single_cell_memmap_dataset.py

```
@@ -360,6 +561,87 @@ def get_row(
             return ret, self._feature_index.lookup(index, select_features=feature_vars)[0]
         else:
             return ret, None
+    
+    # FOR POLINA: this can be separate function or integrated into get_row
+    # FOR POLINA: how might your rowfeatureindex approach be relavant here? 
+    def get_row_with_neighbor(
```

I think if it can return with or without a neighbour, it would make sense for this to be part of get_row. Also, if including the neighbour is passed in but there is no neighbour, it should raise an error.